ExceptionsLogging/NotParsedElementsLog.txt

```diff
@@ -0,0 +1,12 @@
+Could not parse processor. Processor ID is:   6
+Time:    11/21/2020 7:11:50 PM
+Could not parse processor. Processor ID is:   7
+Time:    11/21/2020 10:20:59 PM
+Could not parse processor. Processor ID is:   8
+Time:    11/21/2020 10:35:03 PM
+Could not parse processor. Processor ID is:   9
+Time:    11/21/2020 11:02:27 PM
+Could not parse processor. Processor ID is:   10
+Time:    11/21/2020 11:41:51 PM
+Could not parse processor. Processor ID is:   11
+Time:    11/21/2020 11:47:53 PM
```

Comment:
you should delete this file, cause there are only testing data now

Reply:
Deleted